src/acore-wp-plugin/src/Components/UserPanel/Pages/ItemRestorationPage.php

```
@@ -1,198 +1,292 @@
+                                $clsStyle = AcoreCharColors::rowStyle(intval($char['class']), intval($char['race']));
+                            ?>
+                                <li>
+                                    <button type="button" class="acore-char-row acore-char-card"
+                                         data-char-guid="<?= intval($char['guid']) ?>"
+                                         data-char-name="<?= esc_attr($char['name']) ?>"
+                                         title="<?= esc_attr(sprintf('Show items that can be restored to %s', $char['name'])) ?>"
+                                         style="<?= esc_attr($clsStyle) ?>">
+                                        <span class="acore-char-name"><?= esc_html($char['name']) ?></span>
+                                        <span class="acore-char-meta">
+                                            <span class="acore-level" data-exp="<?= AcoreCharColors::expansionSlug(intval($char['level'])) ?>" title="<?= esc_attr(AcoreCharColors::expansionLabel(intval($char['level']))) ?>">Level <?= intval($char['level']) ?></span>
+                                            <img class="race-icon" height="32" width="32" title="<?= esc_attr(AcoreCharColors::getRaceName(intval($char['race']))) ?>" src="<?= ACORE_URL_PLG . 'web/assets/race/' . intval($char['race']) . (intval($char['gender']) == 0 ? 'm' : 'f') . '.webp' ?>">
```

PHPStan (2.2.2)

[error] Constant ACORE_URL_PLG not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)